docker/build/devdeps.Dockerfile

```diff
@@ -133,7 +133,9 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
         git ninja-build \
         python3 python3-pip libpython3-dev \
         libblas-dev \
-    && python3 -m pip install --no-cache-dir lit pytest numpy \
+    && python3 -m pip install --no-cache-dir \
+        lit pytest numpy \
```

Comment:
can you add `openfermionpyscf` here? That will enable the chemistry tests.

Reply:
Yes. Is this a test only dependency or is this a runtime dependency that needs to be included in our packages?

Reply:
The generation of molecular hamiltonians has been abstracted out into its own interface. We could have multiple plugins that implement that interface. We have one currently which delegates to OpenFermion-PySCF. In this sense, `openfermionpyscf` is not a strict runtime dependency - one could use CUDA Quantum and not require it.

Reply:
I see that the python code is there and raises an error if openfermionpyscf is not installed. That seems clear enough to me, and I would expect that the people who want to use this library probably have that library installed already. I am inclined to not include it in the docker image for now, but that means that people need to install it in the container themselves.